Client/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Zeebe.Client.Builder;
+
+namespace Zeebe.Client
+{
+
+    public static class ServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds the Zeebee builders to the ServicesCollection
+        /// </summary>
+        /// <param name="services"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddZeebeBuilders(this IServiceCollection services)
```

Comment:
Interesting, this is where the builders then registered, right?

Reply:
yes